modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path

```diff
@@ -365,6 +371,93 @@
 	<td></td>
 </tr>
 
+<!--TABS-->
+<tr>
+	<td>RULES_TAB</td>
+	<td>//button[@id='rules']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FIELDS_TAB</td>
+	<td>//button[@id='fields']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_NAME_TEXT_FIELD</td>
+	<td>//input[@placeholder='Untitled Rule']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_ADD_BUTTON</td>
```

Comment:
Hi @calvinctvo , I think this selector should be scoped to the Fields/Rules right sidebar because I just executed the script and I noticed the left sidebar (related to Objects) is opening. Could you please take a look?